app/views/pregnancies/edit.html.erb

```diff
@@ -16,7 +16,7 @@
       <%= render partial: 'pregnancies/patient_information', locals: { f: f, pregnancy: @pregnancy } %>
       <%= render partial: 'pregnancies/change_log', locals: { f: f, pregnancy: @pregnancy } %>
       <%= render partial: 'pregnancies/call_log', locals: { pregnancy: @pregnancy } %>
-      <% end %>
+<% end %>
```

Comment:
This use of blocks is kind of a crime against programming that I am guilty of perpetrating, haha. I'm going to make a note to hack on it and see if we can do it in a less crime-against-programming kinda way, because it is truly awful.